keyboards/cannonkeys/satisfaction75/config.h

```diff
@@ -46,6 +46,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* define if matrix has ghost */
 //#define MATRIX_HAS_GHOST
 
+#define ENCODER_RESOLUTION 2
```

Comment:
Im fairly certain this depends on installed encoder, @awkannan whats your thoughts?
```suggestion
```

Reply:
Oh, ok. I tuned this to the encoder that shipped with the pcb itself: EC11E18244A5